projects/schematics/src/migrations/3_0/constructor-deprecations/data/user-register.effect.migration.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  SPARTACUS_CORE,
+  NGRX_STORE,
+  AUTH_SERVICE,
+  ACTIONS,
+  USER_CONNECTOR,
+  USER_REGISTER_EFFECT,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const USER_REGISTRATION_EFFECT_MIGRATION: ConstructorDeprecation = {
```

Comment:
Let's call it USER_REGISTER_EFFECT_MIGRATION

Reply:
Done